packages/dso-toolkit/src/components/logo/logo.logos.ts

```diff
@@ -0,0 +1,8 @@
+export const logo = `
```

Comment:
Ik wil het logo met inline SVG gaan renderen:

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/d00f229e-91ef-4b99-95ea-f55b27bd28fc)

De tekst "Regels op de kaart" is daarmee ook toegankelijk en selecteerbaar:

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/d8b64776-4940-44b1-b408-f9fbd707e8d0)

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/f9b63a39-f1b2-4ad4-826a-603554fac62e)


Aandachtspunten zijn de viewBox en width van de SVG.

De tekst "Omgevingsloket" wil ik ook in een `<text>` element plaatsen, en inkleuren met styling.